fortuna/src/backend/routes/middleware/auth.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require('jsonwebtoken');
+const config = require('config');
+
+module.exports = function(req, res, next){
```

Comment:
In general it's cleaner to declare the function earlier (something like `function auth(req: typeOfReq, res: typeOfRes, next: typeOfNext): functionReturnType {`

and then have `export default auth;` here. But either way, it is really helpful for everyone later on if you label the types of what this function takes in and returns at least.